Podfile

```diff
@@ -10,7 +10,7 @@ def wordpress_authenticator_pods
   ## ====================
   ##
   pod 'Gridicons', '~> 1.0'
-  pod 'WordPressUI', '~> 1.5.2'
+  pod 'WordPressUI', '~> 1.5.0-beta'
```

Comment:
I know I approved this PR, but I forgot to ask - why the downgrade here? The latest WPUI version is `1.5.4-beta.3`. Why isn't this `1.5.4`?

Reply:
Oh, I see your point. The `~>` tells CocoaPods "go find the most up-to-date in the 1.5.x line" when doing `bundle exec pod install`, which is 1.5.4-beta.3. But this does look like a downgrade because the code is read statically. I'll change it to 1.5.4-beta 👍